test/langtools/jdk/javadoc/doclet/testSpecTag/TestSpecTag.java

```diff
@@ -23,7 +23,7 @@
 
 /*
  * @test
- * @bug 6251738 8226279 8297802 8296546 8305407
+ * @bug 6251738 8226279 8297802 8305407
```

Comment:
Nice catch! It seems this bug id was added by mistake, it does not match the id of the JBS issue where it was added.